### PR TITLE
Add a reveal toggle to the lock screen password field

### PR DESCRIPTION
**Pull request by thooams**
## Problem
The lock screen password field only ever shows dots. There's no way to check what you typed before submitting, so a single typo means guessing again -- more annoying the longer/more complex the password is.

## Solution
- Add an eye button pinned inside the field (to the left of the fingerprint icon, when one is configured) that toggles the field between masked and plain text.
- Add `Ctrl+Space` as a mouse-free equivalent.
- The toggle stays visible but dimmed while the field is empty (nothing to reveal yet), and brightens once you start typing.
- Revealing resets to masked whenever the field is cleared (Escape, Ctrl+U, or a submit attempt), so it never carries over into the next attempt.
- The fingerprint-icon reserved-space math is untouched (`fingerprintReserve`); the eye toggle gets its own `eyeReserve`, and both are summed for the field's margins.

## Why `Ctrl+Space`
There's no established convention here -- no OS, browser, or password manager agrees on one shortcut for this. I deliberately avoided an Alt-based chord: Alt (and AltGr specifically) composes special characters on many non-US keyboard layouts, French AZERTY among them, so an Alt combo risked colliding with that.

## Screenshots

Masked (default) | Revealed (toggled)
:---:|:---:
masked | revealed

## Testing
- `qmllint shell/plugins/lock/LockView.qml`
- `test/shell.d/lock-fingerprint-indicator-test.sh` and `test/shell.d/lock-password-overflow-test.sh` (both exercise the reserved-space math this change also touches)
- Full `test/shell` (185 files) and `test/cli` (116 files) suites -- the only 3 shell-suite failures need a sibling `omarchy-pkgs` checkout and are unrelated to this change
- Manually verified with `omarchy-shell lock preview` and a real `omarchy system lock` session

### CODE REVIEW

File: shell/plugins/lock/LockView.qml [hunk 1]
```diff
@@ -133,26 +149,31 @@ Item {
...
-        anchors.leftMargin: inputField.borderLeft + 18 + root.fingerprintReserve
+        anchors.leftMargin: inputField.borderLeft + 18 + root.fingerprintReserve + root.eyeReserve
         verticalAlignment: TextInput.AlignVCenter
         horizontalAlignment: TextInput.AlignHCenter
         activeFocusOnPress: true
         clip: true
         enabled: root.inputEnabled && !root.authenticatingPassword
         readOnly: root.authenticatingPassword
-        echoMode: TextInput.Password
+        echoMode: root.passwordRevealed ? TextInput.Normal : TextInput.Password
```
[the remark this answers is not in the document]
Comment (thooams) [reply shell/plugins/lock/LockView.qml:171]: Added `test/shell.d/lock-password-reveal-test.sh` (+ fixture) in 1b1546aa, covering: both toggle paths (now unified through a shared `toggleReveal()`), the empty-field no-op, reset-on-clear, and revealed-text overflow scaling.

While adding that test I also fixed the bug you flagged at line 206 (Ctrl+Space had no empty-field guard, unlike the mouse button) and added Accessible.role/name/description/onPressAction at line 251 -- both folded into the same commit since the shared `toggleReveal()` needed for testability was also the natural fix for the Ctrl+Space gap.

**Comment by thooams** [comment]
Added a tooltip (17bed056) on the reveal toggle, surfacing "Show/Hide password · Ctrl+Space" on hover -- the shortcut previously had no on-screen hint. Reused `qs.Ui`'s existing `PanelToolTip` (already the established hover-tooltip pattern elsewhere in the shell) rather than hand-rolling one.
